src/main/java/com/software/modsen/rideservice/dto/response/RideResponseSet.java

```diff
@@ -0,0 +1,16 @@
+package com.software.modsen.rideservice.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Set;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class RideResponseSet {
+    Set<RideResponse> rideResponseSet;
+}
```

Comment:
It's better to rename the field. For example, `items`

Reply:
Renamed